### DISCUSSION TITLE
Enable HTML check in markdownlint-cli2

### DISCUSSION THREAD

**Discussion by myrthos** [gethinode/hinode#466] [Ideas]
Now that unsafe HTML is disabled, would it be an idea to modify `markdownlint-cli2.jsonc` to enable MD033 (which checks for inline HTML). If people do not want it they can add `<!-- markdownlint-disable MD033 -->` in the markdown file or modify the `markdownlint-cli2.jsonc` file themselves.
Might need some documentation as well probably.

**Comment by markdumay** [comment]
Yes, makes sense. We should probably also update the coding guidelines. The docs still use HTML though.

**Comment by myrthos** [comment]
Yes, I noticed the Typography section in the docs also still mentions using HTML.

As I already implemented the change and was removing all HTML from my markdown files found by lint, I noticed it generates errors for `README.md` file because of the HTML used for the badges. That would either require an exclusion of MD033 in the file itself, or perhaps even excluding the file from being linted. Or using markdown instead of HTML.

**Reply by markdumay** [reply]
We could disable the badges in the readme with a comment. Lets create a feature request for the docs- it would be nice not to have to rely on HTML at all. It would be a nice litmus test. I added the card group to improve the about section of the main repo accordingly.

**Reply by myrthos** [reply]
I think the badges can also be made with only markdown, which would be a much nicer solution.

**Comment by markdumay** [comment]
[![Star on GitHub](https://img.shields.io/badge/GitHub-Star-blue?logo=github)](https://github.com/badges/shields)

yes, you’re right!

not sure if we center and horizontally stack it though.

**Comment by myrthos** [comment]
It stacks horizontally. Also when each badge is placed on a new line.  
I found out it doesn't stack when I comment one of the badges out using `<!--  -->`. In that case the next badge is placed on a new line and it stacks again horizontally from there.

It also doesn't work when using a table with a single centered column, because in markdown a table is as wide as the elements in it. So it would look centered on a phone, but not on a desktop. On top of that there is also an extra horizontal line, because of the table header, which is either below or above the badges (depending on where the badges are placed in the table).  
Also the badges cannot be on separate lines anymore and they all need to be on one single line and that just doesn't look good in the editor.

I don't think there is another option in GFM to center something, so if that is required, using HTML is the only option.

I have modified my `README.md` and just left the badges not centered.
